services/server/server.go

```diff
@@ -440,6 +441,13 @@ func (s *Server) validateChangeSettingsRequest(ctx context.Context, req *serverp
 		}
 	}
 
+	if req.DisableStt && req.EnableStt {
+		return status.Error(codes.InvalidArgument, "Enable STT and disable STT cannot be both true")
+	}
+	if req.EnableStt && req.DisableTelemetry {
+		return status.Errorf(codes.InvalidArgument, "Cannot enable STT while disabling telemetry")
+	}
+
```

Comment:
@percona-csalguero @BupycHuk why we validate it there and in models?

Reply:
You are right. I am going to remove it from here.